Packs/CortexResponseAndRemediation/Playbooks/playbook-CVE-2025-49704_and_CVE-2025-49706_and_CVE-2025-53770_and_CVE-2025-53771_-_Microsoft_SharePoint_ToolShell_vulnerability_chain.yml

```
@@ -0,0 +1,3317 @@
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 9604649d-96ad-4e7c-8b1a-00e286ce1981
+    type: title
+    task:
+      id: 9604649d-96ad-4e7c-8b1a-00e286ce1981
+      version: -1
+      name: Investigation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "11"
+      - "12"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 4191.5,
+          "y": 1105
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: 0a4820ee-fc2f-4843-87b1-ee4e5e124d47
+    type: title
+    task:
+      id: 0a4820ee-fc2f-4843-87b1-ee4e5e124d47
+      version: -1
+      name: Hunt For IOCs
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "15"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 2771,
+          "y": 1283
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: b91e46f0-2c7a-4ab9-83aa-018592b4ebbf
+    type: title
+    task:
+      id: b91e46f0-2c7a-4ab9-83aa-018592b4ebbf
+      version: -1
+      name: Anlysis Endpoints
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "133"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 4191.5,
+          "y": 1223
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "15":
+    id: "15"
+    taskid: 239d8f48-891c-4520-88b2-47c7bf73e039
+    type: title
+    task:
+      id: 239d8f48-891c-4520-88b2-47c7bf73e039
+      version: -1
+      name: Collect Indicators
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "124"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 2771,
+          "y": 1620
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 6a6ce6ff-dd80-4358-94fb-bb84445660b6
+    type: regular
+    task:
+      id: 6a6ce6ff-dd80-4358-94fb-bb84445660b6
+      version: -1
+      name: Extract indicators from data collected
```

I am pretty sure all task names should be "Title Case" unless they are in the form of a question. If they are in the form of a question, they can be "Sentence case." 

```suggestion
      name: Extract Indicators from Collected Data
```